developers/uploader/README.md

```diff
@@ -0,0 +1,61 @@
+# Uploader
+
+### What's Uploader?
+
+The Uploader represents a cutting-edge solution designed to streamline the storage management process within a decentralized network. Built with efficiency and scalability in mind, Uploader leverages advanced technologies to provide secure, reliable, and cost-effective storage solutions to users.
+
+### Architecture Overview
```

Comment:
We need a visual for the architecture

Reply:
@AnaLoznianu are these being discuss with the designer too?

Reply:
Not at the moment, if you can create a draft I'll send it to him

Reply:
sounds good, I'll come up with something 👍🏽